components/search-results/header.js

```diff
@@ -18,24 +19,30 @@ export default function SearchResultsHeader({
   const { totalPages } = useSearchLimits(searchResults);
 
   return (
-    <div className="sm:mb-1 sm:flex sm:items-center sm:justify-between">
-      <div className="mb-1 flex gap-1 sm:mb-0">
-        <ViewSwitch searchResults={searchResults} />
-        {columnSelectorConfig && (
-          <ColumnSelector
-            searchResults={searchResults}
-            onChange={columnSelectorConfig.onColumnVisibilityChange}
-            onChangeAll={columnSelectorConfig.onAllColumnsVisibilityChange}
-          />
-        )}
-      </div>
-      <div className="sm:flex sm:items-center sm:gap-1">
-        {totalPages > 1 && <SearchPager searchResults={searchResults} />}
+    <>
+      <div className="sm:mb-1 sm:flex sm:items-center sm:justify-between">
+        <div className="mb-1 flex gap-1 sm:mb-0">
+          <ViewSwitch searchResults={searchResults} />
+          {columnSelectorConfig && (
+            <>
+              <ColumnSelector
+                searchResults={searchResults}
+                onChange={columnSelectorConfig.onColumnVisibilityChange}
+                onChangeAll={columnSelectorConfig.onAllColumnsVisibilityChange}
+              />
+              <DownloadTSV />
+            </>
+          )}
+        </div>
+
         <div className="mb-1 flex items-center gap-1 sm:mb-0">
           <ItemsPerPageSelector />
         </div>
       </div>
-    </div>
+      <div>
```

Comment:
There’s no gap between the pager and the report table below. You can add a `mb-1` to this div to make that gap the same as the gap between the pager and the dropdown above it.